protocol/v2/ssv/validator/signature_verifier.go

```diff
@@ -0,0 +1,44 @@
+package validator
+
+import (
+	"fmt"
+
+	spectypes "github.com/bloxapp/ssv-spec/types"
+	"github.com/cornelk/hashmap"
+
+	"github.com/bloxapp/ssv/operator/keys"
+)
+
+type SignatureVerifier struct {
+	operatorIDToPubkeyCache *hashmap.Map[spectypes.OperatorID, keys.OperatorPublicKey]
```

Comment:
I created a task about combining this and msg validation's cache, the task also includes cleaning this cache ( which doesn't happen here ).
since we don't have huge amounts of operators then its not too bad to even keep all of them here forever (maybe even more efficient)